app/src/lib/design-system-v2/components/Button/Button.scss

```diff
@@ -0,0 +1,142 @@
+.rq-custom-btn.ant-btn {
```

Comment:
![CleanShot 2024-09-11 at 13 59 15](https://github.com/user-attachments/assets/b77b15d8-4988-42c5-9fa0-c1fcbf9ace5f)

Hotkey text styling doesn't look good. It should be has less opacity


![CleanShot 2024-09-11 at 14 00 51@2x](https://github.com/user-attachments/assets/52e1405e-9bb5-4553-9a23-6e4d802d815c)